spec/models/fee_calculator_spec.rb

```diff
@@ -59,6 +59,50 @@
       it { is_expected.to eq(400) }
     end
 
+    context '12時間以内1200円' do
+      let(:attr) do
+        {
+         'type' => 'within',
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.